ansible/playbook/README.adoc

```diff
@@ -147,6 +152,83 @@ For instance, to undo the previous host operation:
 ansible-playbook ansible/playbook/passstore_manage_host_groups.yml -e operation=remove -e group_name=k8s_115 -e vm_name=n01-k115
 ```
 
+=== update_letsencrypt_certificate
+
+[.lead]
+Updates the certificates for the openshift ingress console.
+
+.Playbook parameters
+[%header,cols="25%,75%"]
+|===
+| Variable | Description
+
+| `cluster_type`
+
+[.fuchsia]#string#
+
+a| Type of cluster where the certificate will be deployed into (target).
+
+* *kubernetes*
+* *openshift => Default*
+
+| `site_name`
+
+[.fuchsia]#string# / [.red]#required# 
+
+a| Name of the site that will be used to fetch the certificates from.
```

Comment:
This wording is a bit confusing: "Name of the site". What is a site ? Is it a domain name, something else ?

Reply:
That's a tough one, it's the name of the prefix given to the TLS secret which coincides with the OCP cluster name.

```bash
$ kc --namespace snowdrop-site get secret
...
qshift-snowdrop-dev-tls         kubernetes.io/tls   2      96d
www-snowdrop-dev-tls            kubernetes.io/tls   2      68d
```

Reply:
Ok. That corresponds to a TLS secret which has been created part of a namespace. 

Historically the certificate's related stuffs have been created under 2 namespaces using an ansible playbook: `snowdrop-site` and `halkyon-site`.

I'm not against the fact to keep this convention and that we create additional secrets, certificates request under the namespace which match a DNS domain (example: snowdrop-site => snowdrop.dev) BUT that should be clear to the user and that they know which namespace they should use. As we only manage one domain name, we could set as default `snowdrop-site` to avoid issues. WDYT ? @jacobdotcosta